com.unity.netcode.gameobjects/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -462,15 +462,15 @@ public void TrackPacketReceived(uint packetCount)
 #endif
         }
 
-        public void TrackRttToServer(int rtt)
+        public void TrackRttToServer(int rttMilliseconds)
         {
 #if MULTIPLAYER_TOOLS_1_0_0_PRE_7
             if (!CanSendMetrics)
             {
                 return;
             }
-
-            m_RttToServerGauge.Set(rtt);
+            var rttSeconds = rttMilliseconds * 1e-3;
+            m_RttToServerGauge.Set(rttSeconds);
```

Comment:
@Rosme, you approved my PR to report all metrics in base units. It was discussed, as among the tools team.

Reply:
https://github.com/Unity-Technologies/com.unity.multiplayer.tools/pull/247

Reply:
This is not a workaround, this is how reporting stats works now since PR 247. The metric is declared like this:
```
        [MetricMetadata(
            DisplayName = "RTT To Server",
            MetricKind  = MetricKind.Gauge,
            Units       = Units.Seconds)]
        RttToServer                  = (MT.RttToServer          << NDC.k_BitWidth) | ND.SentAndReceived,
```

Reply:
Here's a link to a Slack post about this as well: https://unity.slack.com/archives/C02A116CT96/p1646941733530899

Reply:
Let me know if you would like to jump on a call about this

Reply:
Well I'll be damned. All good then. I forgot I had approved that